nilmtk/losses.py

```diff
@@ -16,6 +16,15 @@ def f1score(app_gt, app_pred):
 
     return f1_score(gt_temp, pred_temp)
 
+def mcc(app_gt, app_pred):
+    # https://scikit-learn.org/stable/modules/generated/sklearn.metrics.matthews_corrcoef.html?highlight=mcc
+    threshold = 10
+    gt_temp = np.array(app_gt)
+    gt_temp = np.where(gt_temp<threshold,0,1)
+    pred_temp = np.array(app_pred)
+    pred_temp = np.where(pred_temp<threshold,0,1)
+    return matthews_corrcoef(gt_temp, pred_temp)
```

Comment:
`matthews_corrcoef` is missing from the imports in this file, right?

Reply:
You are right. I will resolve this problem. Thanks!